build.gradle

```diff
@@ -40,6 +40,7 @@ dependencies {
 	// -- OpenAPI CodeGen dependencies --
 	implementation group: 'io.swagger.core.v3', name: 'swagger-annotations', version: '2.1.7'
 	implementation group: 'io.springfox', name: 'springfox-swagger2', version: '3.0.0'
+	implementation 'junit:junit:4.13.1'
```

Comment:
I saw we are using junit jupiter,  I was just getting the unit test working late yesterday

Reply:
Yes, you shouldn't need to add a new dependency for your tests. If you do need to add a dependency for a test, remember to use `testImplementation` so its use is scoped to test code only.